packages/webpack-rtl-plugin/test/index.js

```diff
@@ -183,15 +182,15 @@ describe( 'Webpack RTL Plugin', () => {
 		} );
 
 		it( 'should create a two css bundles', () => {
-			expect( fs.existsSync( cssBundlePath ) ).to.be.true;
-			expect( fs.existsSync( rtlCssBundlePath ) ).to.be.true;
+			expect( fs.existsSync( cssBundlePath ) ).toBe( true );
+			expect( fs.existsSync( rtlCssBundlePath ) ).toBe( true );
 		} );
 
 		it( 'should create a second bundle with a different hash', () => {
 			const cssChunk = cssBundleName.split( '.' );
 			const rtlCssChunk = rtlCssBundleName.split( '.' );
 
-			expect( cssChunk[ 1 ] ).to.not.equal( rtlCssChunk[ 1 ] );
+			expect( cssChunk[ 1 ] ).not.toEqual( rtlCssChunk[ 1 ] );
```

Comment:
`.toBe` is enough to compare strings, `.toEqual` is deep comparison.

Reply:
Fixed